wsl-pro-service/internal/system/system_test.go

```diff
@@ -22,6 +22,41 @@ const (
 	mockBadOutput                      // A mock that returns a bad value with no error
 )
 
+func TestNew(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		systemLandscapeConfigFile string
+	}{
+		"Return a new system": {},
+
+		"Only prints a warning when the landscape config validation failed": {systemLandscapeConfigFile: "invalid_ini.conf"},
+	}
+
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
+
+			mock := &testutils.SystemMock{
+				FsRoot: testutils.MockFilesystemRoot(t),
+			}
+
+			if tc.systemLandscapeConfigFile != "" {
+				config, err := os.ReadFile(filepath.Join("testdata", "landscape.conf.d", tc.systemLandscapeConfigFile))
+				require.NoError(t, err, "Setup: could not load fixture")
+				err = os.MkdirAll(filepath.Dir(mock.Path(system.LandscapeConfigPath)), 0700)
+				require.NoError(t, err, "Setup: could not create landscape config dir")
```

Comment:
nit: I think we should capitalize the word "Landscape" in comments and logs/strings.